releases/v2.1.0-alpha.20180730.md

```diff
@@ -0,0 +1,263 @@
+- Added support for signing server and client certificates by different CAs. [#27636][#27636] {% comment %}doc{% endcomment %}
+
+### Enterprise Edition Changes
+
+- Core dumps are now disabled and [encryption](../v2.1/encryption.html) is enabled. [#27426][#27426]
+- [`CHANGEFEED`](../v2.1/change-data-capture.html)s now use an asynchronous Kafka producer, increasing throughput. [#27421][#27421]
+
+### SQL Language Changes
+
+- CockroachDB now supports custom frame specification for [window functions](../v2.1/window-functions.html) using `ROWS` (fully-supported) and `RANGE` modes. For `RANGE`, `<value> PRECEDING` and `<value> FOLLOWING` are not supported. [#26666][#26666] {% comment %}doc{% endcomment %}
+- The `SNAPSHOT` [isolation level](../v2.1/transactions.html#isolation-levels) has been removed. Transactions that request to use it are now mapped to `SERIALIZABLE`. [#27040][#27040]. {% comment %}doc{% endcomment %}
+- When the cost-based optimizer is enabled, it will also affect prepared queries. [#27034][#27034]
+- Upon failing to gather data from other nodes, the [`SHOW CLUSTER QUERIES`](../v2.1/show-queries.html) and [`SHOW CLUSTER SESSIONS`](../v2.1/show-sessions.html) statements now report the details of the error. [#26821][#26821]
+- Improved the description for the `age()` [built-in function](../v2.1/functions-and-operators.html). [#27082][#27082] {% comment %}doc{% endcomment %}
+- The `pg_get_indexdef()` built-in function now supports 3 arguments. [#27161][#27161] {% comment %}doc{% endcomment %}
+- Added `COPY` support to `IMPORT .. . PGDUMP`. [#27062][#27062] {% comment %}doc{% endcomment %}
+- The new `max_row_size` option overrides default limits on line size for `IMPORT ... PGDUMP` and `PGCOPY`. [#27062][#27062] {% comment %}doc{% endcomment %}
+- The `SHOW TRACE FOR <stmt>` statement was incomplete and incorrect and has thus been removed. To use tracing, use `SET tracing = ...` and `SHOW SESSION TRACE` instead. [#26729][#26729] {% comment %}doc{% endcomment %}
+- `SET tracing` accepts a new option `results`, which causes result rows and row counts to be copied to the session trace. This was previously implicit with option `kv` but must now be specified explicitly when desired. [#26729][#26729]
+- The word `view` is now supported as an identifier like in PostgreSQL. [#27204][#27204] {% comment %}doc{% endcomment %}
+- `IMPORT ... PGDUMP` no longer requires the `--no-owner` flag. [#27268][#27268] {% comment %}doc{% endcomment %}
+- `AS OF SYSTEM TIME` can now use some more complex expressions to compute the desired timestamp. [#27206][#27206] {% comment %}doc{% endcomment %}
+- Added support for the `convert_from()` and `convert_to()` built-in functions, for compatibility with PostgreSQL.  For `convert_from()`, however, in contrast with PostgreSQL, the function in CockroachDB accepts `NUL` in the input, because null characters are valid in CockroachDB strings. [#27328][#27328] {% comment %}doc{% endcomment %}
```

Comment:
The word "NUL" is not a reference to code. It is [a proper noun](https://en.wikipedia.org/wiki/Null_character). Therefore it should not be enclosed in code backticks.

Reply:
Done.